pysetup/spec_builders/eip7594.py

```diff
@@ -12,12 +12,41 @@ def imports(cls, preset_name: str):
         return f'''
 from eth2spec.deneb import {preset_name} as deneb
 '''
-    
+
+
+    @classmethod
+    def classes(cls):
+        return f'''
+class PolynomialCoeff(list):
```

Comment:
Interesting approach.

Before merging, is it possible to add a small comment in the 4844 and 7594 poly commitments file that mention that these types can be found in the spec builder (ideally with a permalink)?

I think having them in the spec builder is a plausible tradeoff, but I think people who go over the spec and they are not consensus-specs experts, they would never find those types.